### PR TITLE
Coolcud/palindrome number

### PR DESCRIPTION
**Pull request by Coolcud**
Palindrome number function and 3 tests added

### CODE REVIEW

File: functions/palindrome_number.py [hunk 1]
```diff
@@ -1,2 +1,13 @@
 def palindrome_number(x):
-    pass
```
Comment (hawktalk25) [inline functions/palindrome_number.py:2]: Should remove this when implementing function

Comment (MDHowden2020) [reply functions/palindrome_number.py:2]: I agree!

File: functions/palindrome_number.py [hunk 2]
```diff
@@ -1,2 +1,13 @@
...
+    temp = x
+    reverse = 0
+
+    while (x > 0):
```
Comment (MDHowden2020) [inline functions/palindrome_number.py:5]: Why are there parenthesis here? What were you trying to do?

**Review by MDHowden2020 — COMMENTED** [review]
Sage, I've seen you do better lol.

File: functions/palindrome_number.py [hunk 3]
```diff
@@ -1,2 +1,13 @@
...
+        dig = x % 10
+        reverse = reverse * 10 + dig
+        x = x // 10
```
Comment (hawktalk25) [inline functions/palindrome_number.py:8]: temp is initialized but is not modified in the function

**Review by hawktalk25 — COMMENTED** [review]
Overall, good job! Need to work harder on your typing speed!

File: functions/palindrome_number.py (hunk 3 above)
Comment (Coolcud) [reply functions/palindrome_number.py:8]: temp does not need to be modified, but I realize that temp is not a clear variable name.

File: functions/palindrome_number.py (hunk 2 above)
Comment (Coolcud) [reply functions/palindrome_number.py:5]: they have been removed

File: functions/palindrome_number.py (hunk 1 above)
Comment (Coolcud) [reply functions/palindrome_number.py:2]: was already removed

**Pull request merged**